src/cleanup_rules/swift/rules.toml

```
@@ -51,6 +51,49 @@ groups = ["boolean_expression_simplify"]
 replace_node = "conjunction_expression"
 replace = "@lhs"
 
+#
+# Before 
+#   true || abcd()
+# After 
+#   true
+#
+[[rules]]
+name = "true_or_something"
+query = """(
+(disjunction_expression
+        lhs: [(boolean_literal) @true  
+            (tuple_expression 
+                value: (boolean_literal) @true)]
+        rhs: (simple_identifier) 
```

Here the rhs could be wildcard right?
```suggestion
        rhs: (_) 
```

---

we decided to change only `true || variable` right and not `true || func()` with `true`

---

but `func()` gets short-circuited in this scenario. Here the developer does not expect that `func()` will get invoked if `lhs` is `true`.

---

Okay, I get it. I was thinking if code is something like `TestEnum.stale_flag.isEnabled || func()` and polyglot rules convert it to `true || func()`, Should we remove `func()`.